### DISCUSSION TITLE
`MetaAdam` does not update the model

### DISCUSSION THREAD

**Discussion by xianghang** [metaopt/torchopt#87] [Q&A] [answered]
Hello,

I am trying a simple code like

```python
x_t, y_t = torch.ones(4, 14).int(), torch.ones(4)
net = Dnn(feature_columns)
inner_opt = torchopt.MetaAdam(net, lr=1e-2)
net.train()
for i in range(10):
    loss = loss_func(net(x_t), y_t.view(-1, 1))
    inner_opt.step(loss)
    print(loss)
```
I expect that `inner_opt.step(loss)` would update `net`. However, from the loss values printed out, it seems not be the case.
The loss does not decrease. However, if I replace it with `torch.optim.Adam`, it seems to work fine. 

I meant to have something similar to the example
https://github.com/metaopt/torchopt/blob/main/examples/few-shot/maml_omniglot.py#L155-L158

I am not sure if i have made any mistakes here

Please find this notebook for details and preproduction
https://colab.research.google.com/drive/1h_S0G8ibLFl8B2iRTQ6sIyvr01E3WJbG?usp=sharing

**Comment by XuehaiPan** [comment, marked as answer]
@xianghang Thanks for reporting this. It's a bug and has already been fixed by 714f99d2e2fa122bde620dc7b8a08db41e6c7dfb. New wheels will be released about an hour later.

**Reply by XuehaiPan** [reply]
Fixed by `0.5.0.post4`. Upgrade your installation:

```bash
pip3 install --upgrade torchopt
```

**Reply by xianghang** [reply]
Many thanks
